src/engine/renderer/tr_local.h

```diff
@@ -24,6 +24,8 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
 #ifndef TR_LOCAL_H
 #define TR_LOCAL_H
 
+#include <vector>
```

Comment:
This is included (indirectly) from `Common.h`, the precompiled header that should be included from all files, so you don't have to add it here.

Reply:
Ah, I see, thanks, fixed this too.